users/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
 from django.contrib.auth import views as auth_views
-from .views import register_view
+from .views import register_view, profile_settings_view
 
 app_name = 'users'
 
 urlpatterns = [
     path('register/', register_view, name='register'),
+    path('users/<str:username>/settings', profile_settings_view, name="profile_settings"),
```

Comment:
Any reason for using the `username` as the parameter? Usually it's used the ID of the resource. By using the ID you also shouldn't do any changes if you'd like to remove the username column.